apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -412,6 +419,23 @@ export class SendMessageEmail extends SendMessageBase {
   }
 }
 
+export const createMailData = (options: IEmailOptions, overrides: Record<string, any>): IEmailOptions => {
```

Comment:
```suggestion
export const createMailData = (options: IEmailOptions, overrides: Record<string, string>): IEmailOptions => {
```
Do we have any override that is not string value?

Reply:
Some of them can be undefined, arrays or strings...